pages/docs/tutorials/native/first-kotlin-native-app.md

```diff
@@ -0,0 +1,88 @@
+---
+type: tutorial
+layout: tutorial
+title:  "First Kotlin/Native Application"
+description: "A look at how to compile our first Kotlin/Native application"
+authors: Hadi Hariri 
+date: 2017-12-03
+showAuthorInfo: false
+---
+
+
+In this tutorial we'll see how to
+
+* [Obtain the Kotlin/Native compiler](#Obtaining-the-compiler)
+* [Write the application](#Creating-Hello-Kotlin)
+* [Compile and examine output](#Compiling-and-examining-output)
+* [Run the application](#Running-the-application)
+
+
+## Obtaining the compiler
+
+Kotlin/Native is available for different macOS, Linux and Windows. Depending on the operating system we're working on, we'll need to download
+the correct compiler. While cross-platform compilation is possible (i.e. using one platform to compile for another), in this first tutorial
+we're going to compile for the same operating system we're running on. In our case this will be macOS. 
+
+We can obtain the latest version of the compiler from the [GitHub releases page](https://github.com/JetBrains/kotlin-native/releases).
+
+Once downloaded, we can uncompress it in any folder. For convenience it's useful to add the `bin` folder to the system path so that we can invoke the 
+compiler from any location.
+
+## Creating Hello Kotlin
+
+Our first application is going to simply print some text on the standard output. In our case, this will be "Hello Kotlin/Native"
+ 
+We can open up our favorite IDE or editor and write the following code in a file named `hello.kt` 
+
+```kotlin
+fun main(args: Array<String>) {
+    println("Hello Kotlin/Native!")
+}
+```
+
+## Compiling and examining output 
+
+The next step is to compile the application, which will be done using the compiler downloaded in the first step. If we have the `bin` folder
+correctly added to the path, we should be able to invoke the compiler using
+
+`kotlinc hello.kt`
+
+which is telling the compiler to compile the source code `hello.kt`.
+
+The first time the compiler runs, it downloads a list of necessary requirements (thus first run does take longer). If everything runs correctly, the output 
+should be two files  
+
+`hello-world.kt.bc`
```

Comment:
We shall not emphasise .bc files, this is just a bug in compiler that we do not remove intermediate .bc file. Bitcode excursion is neat, but we shall not document this bug.